lib/msal-core/src/utils/WindowUtils.ts

```diff
@@ -57,15 +57,27 @@ export class WindowUtils {
                     href = contentWindow.location.href;
                 } catch (e) {}
 
-                // Don't process blank pages or cross domain
-                if (!href || href === "about:blank") {
-                    return;
-                }
+                if (isSilentCall) {
+                    /*
+                     * Always run clock for silent calls
+                     * as silent operations should be short,
+                     * and to ensure they always at worst timeout.
+                     */
+                    ticks++;
+                } else {
+                    // Don't process blank pages or cross domain
+                    if (!href || href === "about:blank") {
+                        return;
+                    }
```

Comment:
To clarify: was the reason that silent operations weren't timing out was because they'd always return on this statement here and not increment `ticks`?

Reply:
I think we never return from the server domain.

Reply:
@jmckennon Correct.

Reply:
Cool, thanks for the info